src/System.Private.ServiceModel/tools/scripts/sync-pr.cmd

```diff
@@ -60,15 +66,17 @@ if "%3"=="" (
         )
     )
 ) else (
-    set __BRANCH_OR_PR_ID=%3
+    set __BRANCH_OR_PR_ID=%4
     echo   [%~n0] WARNING: This script should usually only be called only from the context of a GitHub Pull Request
-    echo   [%~n0] Branch or PR ID overridden: '%3'
+    echo   [%~n0] Branch or PR ID overridden: '%4'
```

Comment:
There's some expansion funniness, hence why this has to stay as %4 and not _BRANCH_OR_PR_ID. Basically the variable is only defined after exiting the if.